controllers/5-reviews.controllers.js

```diff
@@ -0,0 +1,13 @@
+const { selectReview } = require("../models/5-reviews.models")
+
+exports.manageReviews = (request,response,next) => {
+    const {sort_by, order} = request.query
+   
+    selectReview(sort_by, order).then((result) => { 
+        return response.status(200).send({'reviews': result})
+        
+   })
+   .catch(err => {
+       next(err)
+   })
```

Comment:
Nice work adding this, it will ensure your errors get handled in the correct way using error handling middleware :)